code/datums/keybindings/human.dm

```diff
@@ -42,3 +42,10 @@
 		return
 	var/obj/item/clothing/accessory/holster/H = locate() in M.w_uniform
 	H?.holster_verb()
+
+/datum/keybinding/human/parry
+	name = "Parry"
```

Comment:
I really want this to deafult to F, `keys = list("F")`
 
This is a core combat action, it should be a default one.

F is currently default bound to intent cycling, dunno if we can yeet it. Design team input required.

Code is good apart from that.

Reply:
I think this can be done at a later date. I really want to get this PR in.